Update Gardener docu

vasu1124 opened the pull request:
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [X] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

istio-testing commented:
Hi @vasu1124. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=istio%2Fistio.io).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

windsonsea commented:
/ok-to-test

istio-testing commented:
@vasu1124: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | 99b10a0b581a14cea20b8804c78922deb3a1c82a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12326/lint_istio.io/1599736175055081472) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@vasu1124: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | 71c592fddcfd1b38c4341a071ce777ebe7b3ce5d | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12326/lint_istio.io/1599742792093929472) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

vasu1124 commented:
/retest

kfaseela commented:
/test doc.test.profile_default

berendt commented:
/lgtm

vasu1124 commented:
@kfaseela anything else needed from me?

kfaseela commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
-{{< boilerplate untested-document >}}
+To set up your own [Gardener](https://gardener.cloud) for your organization's Kubernetes-as-a-Service needs, follow the
+[documentation](https://github.com/gardener/gardener/blob/master/docs/README.md).
+For testing purposes, you can set up [Gardener on your laptop](https://github.com/gardener/gardener/blob/master/docs/development/getting_started_locally.md) by checking out the source code repo and simply running `make kind-up gardener-up` (the easiest developer way of checking out Gardener!).
```
source code repository

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
-## Bootstrapping Gardener
+Alternatively, [`23 Technologies GmbH`](https://23technologies.cloud/) offers a fully-managed Gardener service that conveniently works with all supported cloud providers and comes with a free trial: [`Okeanos`](https://okeanos.dev/). Similarly, cloud providers such as [`STACKIT`](https://stackit.de/), [`B'Nerd`](https://bnerd.com/), [`MetalStack`](https://metalstack.cloud/), and many others run G ...
+
+To learn more about the inception of this open source project, read [Gardener Project Update](https://kubernetes.io/blog/2019/12/02/gardener-project-update/), [Gardener - The Kubernetes Botanist](https://kubernetes.io/blog/2018/05/17/gardener/) on [`kubernetes.io`](https://kubernetes.io/blog).
```
```suggestion
To learn more about the inception of this open source project, read [Gardener Project Update](https://kubernetes.io/blog/2019/12/02/gardener-project-update/) and [Gardener - The Kubernetes Botanist](https://kubernetes.io/blog/2018/05/17/gardener/) on [`kubernetes.io`](https://kubernetes.io/blog).
```

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
-    your project. You can skip this step if you create your cluster using the
-    user interface; it is only needed for programmatic access, make sure you set
-    `export KUBECONFIG=garden-my-project.yaml` in your shell.
+    your project. Make sure you set `export KUBECONFIG=garden-my-project.yaml` in your shell.
```
You might replace set with add, or maybe just remove it altogether.
```suggestion
    your project. Make sure you `export KUBECONFIG=garden-my-project.yaml` in your shell.
```

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
-    `export KUBECONFIG=garden-my-project.yaml` in your shell.
+    your project. Make sure you set `export KUBECONFIG=garden-my-project.yaml` in your shell.
     ![Download kubeconfig for Gardener](images/gardener_service_account.png "downloading the kubeconfig using a service account")
+    You can skip this step if you intend to create your cluster using the
```
Does it make sense to move the statement of optionally doing this step to the front of the step?

vasu1124 replied:
thank you. embedded that statement in a <tip/>

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
 
 ### Creating a Kubernetes cluster
 
 You can create your cluster using `kubectl` cli by providing a cluster
```
```suggestion
You can create your cluster using the `kubectl` cli by providing a cluster
```

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
 You can create your cluster using `kubectl` cli by providing a cluster
 specification yaml file. You can find an example for GCP
 [here](https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml).
 Make sure the namespace matches that of your project. Then just apply the
```
```suggestion
Make sure the namespace matches that of your project. Then apply the
```

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
 $ kubectl apply --filename my-cluster.yaml
 {{< /text >}}
 
 The easier alternative is to create the cluster following the cluster creation
```
```suggestion
An easier alternative is to create the cluster following the cluster creation
```

ericvn commented on content/en/docs/setup/platform-setup/gardener/index.md:
```diff
@@ -70,8 +70,8 @@ Gardener dashboard or via cli as follows:
-This kubeconfig file has full administrators access to you cluster. For the rest
-of this guide be sure you have `export KUBECONFIG=my-cluster.yaml` set.
+This kubeconfig file has full administrator access to you cluster.
+For any activities with the payload cluster be sure you have `export KUBECONFIG=my-cluster.yaml` set.
```
Not sure if you need the set. Maybe this (and similar in the prior comment) should be ... exported `KUBECONFIG=my-cluster.yaml`?

vasu1124 replied:
I will leave as is for c&p.

istio-testing commented:
@vasu1124: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | 6ed273add0af7d641b8739c5238a6d68e2611376 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12326/lint_istio.io/1601159007836311552) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@vasu1124: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | 58ec3025d759b8d409b3c6edbb655a8ac7c7a144 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12326/lint_istio.io/1601189508395569152) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

vasu1124 commented:
@ericvn thanks for the review. incorporated the feedback.
@kfaseela cleaned up outdated images, squashed, and final lint completed. ready from my side.

kfaseela commented:
LGTM.. Leaving to Eric for merging

ericvn commented:
/cherry-pick release-1.16

istio-testing commented:
@ericvn: new pull request created: #12351

<details>

In response to [this](https://github.com/istio/istio.io/pull/12326#issuecomment-1344585121):

>/cherry-pick release-1.16


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>
